sql/moz-fx-data-shared-prod/telemetry_derived/firefox_android_clients_v1/init.sql

```diff
@@ -0,0 +1,312 @@
+      (description = "Manufacturer of the device where the client is installed."),
+      device_model STRING
+    OPTIONS
+      (description = "Model of the device where the client is installed."),
+      os_version STRING
+    OPTIONS
+      (description = "Version of the Operating System where the client is originally installed."),
+      adjust_campaign STRING
+    OPTIONS
+      (description = "Structure parameter for the campaign name."),
+      adjust_ad_group STRING
+    OPTIONS
+      (description = "Structure parameter for the the ad group of a campaign."),
+      adjust_creative STRING
+    OPTIONS
+      (description = "Structure parameter for the creative content of a campaign."),
+      adjust_network STRING
+    OPTIONS
+      (description = "The type of source of a client installation."),
+      install_source STRING
+    OPTIONS
+      (description = "The source of a client installation."),
+      metadata STRUCT<
+        reported_first_session_ping BOOL
+        OPTIONS
+          (description = "True if the client ever reported a first_session ping."),
+          reported_metrics_ping BOOL
+        OPTIONS
+          (description = "True if the client ever reported a metrics ping."),
+          min_first_session_ping_date DATE
+        OPTIONS
+          (description = "Date of the earliest first_session ping reported."),
+          adjust_network__source_ping STRING
+        OPTIONS
+          (description = "Name of the ping that reported the first adjust_network value."),
+          install_source__source_ping STRING
+        OPTIONS
+          (description = "Name of the ping that reports the install_source value."),
+          adjust_network__source_ping_date DATE
+        OPTIONS
+          (description = "Date of the ping that reported the first adjust_network value."),
+          install_source__source_ping_date DATE
+        OPTIONS
+          (description = "Date of the ping that reported the first install_source value.")
+      >
+  )
+PARTITION BY
+  first_seen_date
+CLUSTER BY
+  channel,
+  first_reported_country,
+  adjust_network,
+  install_source
+OPTIONS
+  (
+    description = "First observations for Firefox Android clients retrieved from the earliest pings: baseline, first_session and metrics. The attributes stored in this table include the first attribution, device, OS version and ISP. This table should be accessed through the user-facing view `telemetry.firefox_android_clients`. Proposal: https://docs.google.com/document/d/12bj4DhCybelqHVgOVq8KJlzgtbbUw3f68palNrv-gaM/. For more details about attribution and campaign structure see https://help.adjust.com/en/article/tracker-urls#campaign-structure-parameters."
+  );
+
+INSERT
+  analysis.lvargas_firefox_android_clients_v1
+WITH first_seen AS (
+  SELECT
+    client_id,
+    first_seen_date
+  FROM
+    `mozdata.org_mozilla_firefox.baseline_clients_first_seen`
+  WHERE
+    submission_date >= '2021-01-01'
+    AND first_seen_date = submission_date
```

Comment:
What's going on with this statement? Looking over the defn this is only always true for Fenix clients (not migrated Fennec clients)

Reply:
Note that we have duplicate client_ids on this table, and in those cases the same client ids have multiple first_seen_date and submission_date, but they are equal in a given row.

We should fix upstream or accept the duplicates, as they have been generated in error.

Reply:
Hey Frank, could you point me to the definition you mention?
For what I see in the [metadata of the baseline_clients_first_seen table](https://github.com/mozilla/bigquery-etl/blob/afad4d84378266774dcc0d745fda2feea6ed46d3/sql/moz-fx-data-shared-prod/org_mozilla_firefox_derived/baseline_clients_first_seen_v1/metadata.yaml#L7),  migrated clients are not captured in this table.

Reply:
It shouldn't be the case that we have duplicates any longer, see [this code that Kik added](https://github.com/mozilla/bigquery-etl/blob/generated-sql/sql/moz-fx-data-shared-prod/org_mozilla_firefox_derived/baseline_clients_first_seen_v1/query.sql#L109). @bani let us know if you've seen dupes since then - we will try and figure out why they're still happening.

Reply:
Removed to include migrated clients too.